ci-conda.Dockerfile

```
@@ -37,7 +37,8 @@ EOF
 RUN <<EOF
 case "${LINUX_VER}" in
   "ubuntu"*)
-    apt-get update && apt-get install -y wget
+    i=0; until apt-get update -y; do ((++i >= 5)) && break; sleep 10; done
+    apt-get install -y wget
```

```suggestion
    apt-get install -y --no-install-recommends wget
```

Could we please add a `--no-install-recommends` here, to avoid pulling in unnecessary packages?